specification/customer-insights/resource-manager/Microsoft.CustomerInsights/2017-04-26/customer-insights.json

```diff
@@ -65,6 +65,449 @@
         }
       }
     },
+	"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}/SuggestTypeSchema": {
+      "post": {
+        "tags": [
+          "Hubs"
+        ],
+        "operationId": "Hubs_SuggestTypeSchema",
+        "x-ms-examples": {
+          "Hubs_SuggestTypeSchema": {
+            "$ref": "./examples/HubsSuggestTypeSchema.json"
+          }
+        },
+        "description": "Suggest the type schema for the hub.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Hub.",
+            "pattern": "^[a-zA-Z][a-zA-Z0-9]+$",
+            "minLength": 1,
+            "maxLength": 64
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SuggestTypeSchemaActionInput"
+            },
+            "description": "Parameters supplied to the SuggestTypeSchema operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. return the suggested type.",
+            "schema": {
+			  "$ref": "#/definitions/SuggestTypeSchemaActionResult"
+            }
+          }
+        }
+      }
+    },
+	"/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}/CrmMetadata/discoverorganizations": {
+      "get": {
+        "tags": [
+          "CrmMetadata"
+        ],
+        "operationId": "CrmMetadata_GetDiscoverorganizations",
```

Comment:
The operation name should be pascal cased, CrmMetadata_GetDiscoverOrganizations, and camel-cased in the path.